### DISCUSSION TITLE
Testnet Mode not loading wallet

### DISCUSSION THREAD

**Discussion by dalvira** [phantom/docs#229] [Help & Support] [answered]
### Summary

![Screenshot 2024-04-26 at 10 31 00 AM](https://github.com/phantom/docs/assets/2797351/487ffd10-635a-4357-a8e0-865c86bfd718)

When setting Phantom wallet to Solana Devnet in the Developer Settings, the wallet seems to be stuck in a loading state. This just started happening recently. I tried restarting the browser, reinstalling the wallet, and even downloading the mobile version but the issue still persists. What do I need to do to fix this? Thanks.

### Steps to Reproduce

Go to Developer Settings and turn on Testnet Mode. Go back to your wallet.

### Phantom Version

24.6.0

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Hi @dalvira,

Thank you for writing in!

Apologies for the bad experience. We are in the process of removing Goerli since that has been deprecated. Unfortunately it's causing some issues. You can remedy this by either switching to the Sepolia network for your Etherum testnet or disabling Ethereum (if you are working with Solana only) in settings.

**Reply by dalvira** [reply]
Works, thanks @adamdelphantom!
